src/status.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+import { ourStore } from './index';
+/* eslint-enable */
+
+const status = () => {
+  const checks = Object.values(document.getElementsByName('status'));
+  checks.forEach((element, i) => {
+    element.addEventListener('click', () => {
+      if (element.checked) {
+        ourStore[i].completed = true;
+        (element.parentElement.style.textDecoration = 'underline line-through');
+        alert('Task Completed!');
+      } else {
+        console.log(ourStore[i].completed = false);
+        (element.parentElement.style.textDecoration = 'none');
+        alert('Completed Task Cancelled!');
```

Comment:
- [ ] When we marked the checkbox, the data in local storage did not get updated correctly and so when we refresh the page it goes back to the original position. Kindly fix this issue.

- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

Marking the task
![image](https://user-images.githubusercontent.com/53019916/130254984-532def62-b5f6-4ef9-9ec6-cb0f33b751be.png)

Local state after marking
![image](https://user-images.githubusercontent.com/53019916/130255048-844163ff-d551-451f-a7e4-49bbf6dbb774.png)